services/catarse/app/workers/direct_refund_worker.rb

```diff
@@ -10,7 +10,7 @@ def perform(payment_id)
     begin
       BalanceTransaction.insert_contribution_refunded_after_successful_pledged(payment.contribution_id)
 
-      if payment.slip_payment? && payment.paid?
+      if (payment.slip_payment? || payment.pix_payment?) && payment.paid?
```

Comment:
**Dúvida**
Lendo o documento que o Pagarme nos enviou sobre o Pix, notei que eles têm a opção do reembolso já para a conta do usuário, sem precisarmos adicionar o dinheiro no saldo do usuário e pedir pra ele sacar. Não vamos utilizar esse mecanismo?

Reply:
Estou esperando a confirmação no notion:
https://www.notion.so/catarse/Defini-o-sobre-reembolso-de-PIX-e-Boleto-24222c14f6c94bc1b56a0ce082d25540

Reply:
Isso mesmo @gabrielras Reforcei lá o pedido no Notion e se não tivermos uma confirmação da Fê ainda no início da semana, vou considerar a decisão que foi feita lá como a decisão final. Te aviso por lá.